docs/release/release_0_6_1.md

```diff
@@ -15,12 +15,12 @@ For more information, examples, and documentation, please visit our website: htt
 ### The HiLo👋 Colormap!
 
 Introducing the HiLo colormap to napari! 🎨
-This much-loved colormap (LUT) is like grayscale, except it displays values at or above the maximum contrast limit as red 🔴 and values at or below the minimum contrast limit as blue 🔵.
+This much-loved colormap (LUT) visualizes the maximum contrast limit as red 🔴 and the minimum contrast limit as blue 🔵.
```

Comment:
Can you revert this change and the one below in the release-tools additional notes?